fleet-management-persistence/src/main/java/cz/fi/muni/pa165/entity/Employee.java

```diff
@@ -55,6 +64,22 @@ public void setSurname(String surname) {
         this.surname = surname;
     }
 
+    public Set<VehicleCategory> getVehicleCategories() {
+        return vehicleCategories;
+    }
+
+    public void setVehicleCategories(Set<VehicleCategory> vehicleCategories) {
+        this.vehicleCategories = vehicleCategories;
```

Comment:
Write test, if you set new `Categories` to `Employee`, if also the `employee` collection of each `Category` gets updated.